downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -53,8 +53,78 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.4.0](https://git
 
 ### New features
 
+#### Ability to get the strand dump during `bal test`
+When running the tests in a Ballerina package or a file using the `bal test` command, the strand dump can be obtained by sending the `SIGTRAP` signal to that process denoted by the `BTestMain` class-name.
+
 ### Improvements
 
+#### Improvements in Runtime Java APIs
+
+##### Get the user-defined type name on Singleton types
+
+Calling `getName()` on the runtime class `FiniteType` will return the user-defined type name if it is available.
+
+For example, if a constant is defined in the following way, the `getName()` method on the `FiniteType` will return the string `”OPEN”`.
+
+```ballerina
+const OPEN = "open";
+```
+
+##### Type-reference Type Support in Runtime Java APIs
+
+The following runtime APIs are now modified to return type-reference type instances according to the type definitions.
+
+- getElementType - from runtime class `ArrayType`
```

Comment:
```suggestion
- `getElementType` - from runtime class `ArrayType`
```
Add backticks in all the items of the list below. Shall we add this list in a table?